psi4/src/psi4/libmints/cdsalclist.cc

```diff
@@ -121,7 +121,7 @@ CdSalcList::CdSalcList(std::shared_ptr<Molecule> mol, int needed_irreps, bool pr
         double atomx = molecule_->x(i);
         double atomy = molecule_->y(i);
         double atomz = molecule_->z(i);
-        double smass = sqrt(molecule_->mass(i));
+        double smass = sqrt(molecule_->mass(i, false));
```

Comment:
I'm _pretty_ sure this should return zero mass so that COM isn't effected in a nonphysical way, but can I get a second opinion on this one?